app/assets/javascripts/oxalis/model/sagas/save_saga.js

```diff
@@ -166,29 +177,29 @@ export function* sendRequestToServer(
         compress: true,
       },
     );
-    yield put(setVersionNumberAction(version + compactedSaveQueue.length, tracingType));
-    yield put(setLastSaveTimestampAction(undefined, tracingType));
-    yield put(shiftSaveQueueAction(saveQueue.length, tracingType));
-    yield call(toggleErrorHighlighting, false);
+    yield* put(setVersionNumberAction(version + compactedSaveQueue.length, tracingType));
+    yield* put(setLastSaveTimestampAction(undefined, tracingType));
+    yield* put(shiftSaveQueueAction(saveQueue.length, tracingType));
+    yield* call(toggleErrorHighlighting, false);
   } catch (error) {
-    yield call(toggleErrorHighlighting, true);
+    yield* call(toggleErrorHighlighting, true);
     if (error.status >= 400 && error.status < 500) {
       // HTTP Code 409 'conflict' for dirty state
       window.onbeforeunload = null;
       if (error.status === 409) {
-        yield call(alert, messages["save.failed_simultaneous_tracing"]);
+        yield* call(alert, messages["save.failed_simultaneous_tracing"]);
       } else {
-        yield call(alert, messages["save.failed_client_error"]);
+        yield* call(alert, messages["save.failed_client_error"]);
       }
       location.reload();
       return;
     }
-    yield delay(SAVE_RETRY_WAITING_TIME);
-    yield call(sendRequestToServer);
+    yield* call(delay, SAVE_RETRY_WAITING_TIME);
+    yield* call(sendRequestToServer);
   }
 }
 
-export function toggleErrorHighlighting(state: boolean) {
+export function toggleErrorHighlighting(state: boolean): void {
```

Comment:
Isn't `void` the implicit return type?

Reply:
We already talked about this, but for the record: Flow wasn't able to infer this for some reason and complained here.